7_react/react1/week2/src/TodoList/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import TodoItem from "TodoItem/TodoItem"
+import "./TodoList.css"
+
+
+const todoList = [{
+    num: 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017"
+},
+{
+    num: 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017"
+},
+{
+    num: 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017"
+}]
+
+function TodoList(){
+    const todoItemsComponents = todoList.map(item => (
+        <TodoItem key={item.num} description={item.description} deadline={item.deadline}/>
+    ))
+        function App() {
+        const [name,setName] = useState("");
+        var myArray = [];
+        const handleAdd = () => {
+          myArray = [...myArray,name]
+          setName("")
+        }
+        return (
+          <div className="App">
```

Comment:
In React, the App component should always be outside other components, i.e. standing alone. This is because App is the main component that references the other components and acts as an "entry point" or "root" of the application. 

Inside App, you could call the Todo list like this: 
`<TodoList />`

See the first code example here: 
https://react.dev/learn#components